packages/sdk-react/src/api/client.ts

```diff
@@ -54,6 +53,6 @@ export const createAPIClient = ({
 };
 
 export const isMoniteEntityIdPath = (path: string) =>
-  /^\/(?!auth|entities|entity_users\/me|entity_users\/my_entity|entity_users\/my_role|events|mail_templates|webhook_subscriptions|webhook_settings|receivables\/variables|settings|files|mailbox_domains|payable_purchase_orders|frontend|internal)\b/.test(
+  /^\/(?!auth|entities|entity_users\/my_entity|events|mail_templates|webhook_subscriptions|webhook_settings|receivables\/variables|settings|files|mailbox_domains|payable_purchase_orders|frontend|internal)\b/.test(
```

Comment:
Let's not send the Headers which are not needed. I suppose these RegExp parts are still needed:
```
/me|entity_users\/my_entity|entity_users\/my_role
```

Reply:
agreed to fix it in https://monite.atlassian.net/browse/DEV-11719